src/Components/WebAssembly/Build/testassets/standalone/custom-service-worker-assets.js

```diff
@@ -0,0 +1,157 @@
+﻿self.assetsManifest = {
```

Comment:
Not sure why this file is being added to git here, but I don't think it should be.

My guess is that, at some point, there was a mistake and the file was generated in the wrong place (here).

So, can you delete this file?